### PR TITLE
RANCHER 247

### PR DESCRIPTION
**Pull request by DariaPavlova1**
Terraform code to deploy rancher 2.6.5 with a helm chart to the existing EKS cluster.

### CODE REVIEW

**Review by MaryiaLazavik — APPROVED** [review]

**Review by OHaimanov — CHANGES_REQUESTED** [review]
Please move code from terraform/rancher/cluster/rancher_on_eks to terraform/rancher/rancher_server

**Comment by DariaPavlova1** [comment]
> Please move code from terraform/rancher/cluster/rancher_on_eks to terraform/rancher/rancher_server

Moved in c73120db77424d9acd2a86a11a0e6672b3c1c586.

**Review by VitalyDemchenkoEpam — APPROVED** [review]

File: terraform/rancher/rancher_server/variables.tf [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+variable "rancher_server_url" {
+  type        = string
+  default     = "https://rancher.ci.folio.org/v3"
+  description = "Rancher server URL"
+}
+
+variable "rancher_token_key" {
+  type        = string
+  default     = "token-89xnj:9fvk2j8rkmx6mlwwzhzzn96phcf4pssnwszb9kcldzbzxwjgr24zpv"
```
Comment (OHaimanov) [inline terraform/rancher/rancher_server/variables.tf:21]: remove token form PR sensitive data

**Review by OHaimanov — CHANGES_REQUESTED** [review]

File: terraform/rancher/cluster/rancher_on_eks/provider.tf [hunk 2]
```diff
@@ -0,0 +1,41 @@
+terraform {
+  required_providers {
+    kubectl = {
+      source = "gavinbunney/kubectl"
+    }
+  }
+}
+data "aws_eks_cluster" "cluster" {
+  name = "rancher"
```
[the remark this answers is not in the document]
Comment (DariaPavlova1) [reply terraform/rancher/cluster/rancher_on_eks/provider.tf]: Moved in 22edb130434404281611453670c979b39b242a77

File: terraform/rancher/cluster/rancher_on_eks/helm.tf [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+  name             = "rancher"
+  repository       = "https://releases.rancher.com/server-charts/stable"
+  chart            = "rancher"
+  version          = var.rancher_version
+  namespace        = "cattle-system"
+  create_namespace = true
+
+  set {
+    name  = "hostname"
+    value = "rancher.ci.folio.org"
```
[the remark this answers is not in the document]
Comment (DariaPavlova1) [reply terraform/rancher/cluster/rancher_on_eks/helm.tf]: Moved in 22edb130434404281611453670c979b39b242a77

File: terraform/rancher/rancher_server/variables.tf (hunk 1 above)
Comment (DariaPavlova1) [reply terraform/rancher/rancher_server/variables.tf]: Deleted in 22edb130434404281611453670c979b39b242a77

File: terraform/rancher/rancher_server/provider.tf [hunk 4]
```diff
@@ -0,0 +1,54 @@
+terraform {
+  required_providers {
+    kubectl = {
+      source = "gavinbunney/kubectl"
```
Comment (VitalyDemchenkoEpam) [inline terraform/rancher/rancher_server/provider.tf:4]: should a specific version be specified?

**Review by OHaimanov — APPROVED** [review]

**Pull request merged**